src/components/Categories.js

```diff
@@ -1,9 +1,23 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+import Header from './Header';
 
-const Categories = () => (
-  <div className="categories">
-    <button type="button">CHECK STATUS</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  const handleStatusClick = () => {
+    dispatch(checkStatus());
+  };
+
+  return (
+    <div className="categories">
+      <Header />
+      <button type="button" onClick={handleStatusClick}>
+        CHECK STATUS
+      </button>
+    </div>
+  );
+};
 
```

Comment:
- [ ] After dispatching the action to the store we still need to fetch the "Under construction" string and display it on the page. Kindly do that.